tracer/test/test-applications/Samples.Shared/SampleHelpers.cs

```diff
@@ -223,6 +248,17 @@ public static IDisposable GetActiveScope()
             return (IDisposable) ActiveScopeProperty.Invoke(tracer, Array.Empty<object>());
         }
 
+        public static object GetActiveSpanContext()
+        {
+            if (SpanContextProperty is null || SpanProperty is null)
+            {
+                return new NoOpDisposable();
+            }
```

Comment:
We should probably throw here no? This should never happen

Reply:
I agree but I'd like to keep it like that for consistency as we don't throw anywhere in SampleHelpers

Reply:
> we don't throw anywhere in SampleHelpers

I guess that makes sense, as the sample helpers will work in a no-instrumented scenario then